src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -340,32 +127,31 @@ public static long UMod(uint i, uint j)
         // Matching return types of throw helpers enables tailcalling them. It improves performance
         // of the hot path because of it does not need to raise full stackframe.
         //
-
         [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
If you are touching these helpers, you can also delete `[MethodImpl(MethodImplOptions.NoInlining)]` and change them to return `void`.

These are old workarounds from the times when the JIT was not smart enough about throw helpers.